sdcm/utils/url_validate.py

```diff
@@ -0,0 +1,41 @@
+import re
+
+import requests
+
+SCYLLA_URL_REGEX = re.compile('.*(?P<url>http.*scylladb.com.*).*')
+URL_REGEX_AND_FORMAT_LIST = [
+    (re.compile(r'baseurl=(?P<url>http.*scylladb.com.*)'), '{url}repodata/repomd.xml'),
+    (re.compile(r'deb\s+\[arch=(?P<arch>.*?)\]\s(?P<url>http.*?)\s(?P<version_code_name>.*?)\s(?P<component>.*?)$'),
+     "{url}/dists/{version_code_name}/{component}/binary-{arch}/Packages"),
+    (re.compile(r'deb\s+(?P<url>http.*?)\s(?P<version_code_name>.*?)\s(?P<component>.*?)$'),
+     "{url}/dists/{version_code_name}/{component}/binary-amd64/Packages.gz"),
+]
+URL_ENV_VARIABLES = {
```

Comment:
what about `$basearch=noarch`?

Reply:
example: http://downloads.scylladb.com/rpm/unstable/centos/branch-4.0/2020-04-05T12:42:44Z/scylla/noarch/

Reply:
The variable `basearch` can contain more options?